### ISSUE TITLE
Encoded new block can not be decoded with original values

### ISSUE THREAD

**Issue by tomploem** [LiskHQ/lisk-sdk#6043]
### Expected behavior
When using ApiClient, you are able to subscribe to `app:block:new`. This returns the correct datastructure `{ block: "..", accounts: [] }`. A block in its turn be decoded via `client.block.decode(..)` and access for example the current height.

### Actual behavior
`client.block.decode(..)` returns every block value in its defaul values `{version: 0, timestamp: 0, height: 0, , …}`. The value are always defaulting after this._chain.events.on(EVENT_NEW_BLOCK, ...) in lisk-sdk/node encodes the event information:
`
this._channel.publish(APP_EVENT_BLOCK_NEW, {
    block: this._chain.dataAccess.encode(block).toString('hex'),
    accounts: ...
});
`

### Steps to reproduce
We are using with and apiClient in the browser. That originally showcased the height as always 0. The information can also be found by debugging a project using the SDK. 

### Which version(s) does this affect? (Environment, OS, etc...)
 lisk-sdk 5.0.0, chrome, node v12.17.0

**Comment by tomploem** [comment]
This could be related to #6039 but logged separately because it occurs happens in the SDK

**Comment by shuse2** [comment]
@tomploem 
can you try with below code?
```
const client = await lisk.apiClient.createWSClient('ws://localhost:8080/ws');
client.subscribe('app:block:new', ({block}) => { const b = client.block.decode(Buffer.from(block, 'hex')); console.log(b); })
```
Probably it's not clear from the browser without documentation, but the `decode` expects `Buffer` instance for decoding now.

**Comment by tomploem** [comment]
This results in the same error as #6039:

`
Uncaught (in promise) Error: Unsupported data:8,2,16,142,192,200,254,5,24,245,143,1,34,32,22,157,224,68,16,176,133,146,232,181,91,32,90,140,92,227,188,238,135,143,162,28,201,102,123,162,15,206,102,113,101,146,42,32,227,176,196,66,152,252,28,20,154,251,244,200,153,111,185,36,39,174,65,228,100,155,147,76,164,149,153,27,120,82,184,85,50,32,38,118,5,174,35,161,87,176,168,36,83,252,72,221,92,106,220,37,4,92,58,74,42,40,220,101,80,169,81,138,35,147,56,128,202,181,238,1,66,26,8,133,143,1,16,176,143,1,26,16,138,49,167,232,193,28,183,98,236,17,188,115,25,68,255,252,74,64,27,161,0,170,4,165,30,198,215,249,125,176,83,94,114,201,243,57,229,111,48,29,127,207,169,61,168,255,227,52,208,225,47,122,137,77,252,95,77,165,83,248,222,188,157,43,148,64,156,204,89,190,38,168,213,107,120,189,71,211,198,30,136,0 and format:undefined. Currently only Buffers or hex and utf8 strings are supported.
`

**Comment by corbifex** [comment]
> ```
> const client = await lisk.apiClient.createWSClient('ws://localhost:8080/ws');
> client.subscribe('app:block:new', ({block}) => { const b = client.block.decode(Buffer.from(block, 'hex')); console.log(b); })
> ```
> 
> Probably it's not clear from the browser without documentation, but the `decode` expects `Buffer` instance for decoding now.

client.block.decode expects a "normal" buffer which doesn't exist in the browser `Buffer.from(block, 'hex')` returns a Uint8Array. [Buffer vs Uint8Array](https://www.quora.com/What-is-the-relationship-between-a-Buffer-and-an-Uint8Array-in-Node-js/answer/Kai-Zhu-9)

**Comment by shuse2** [comment]
Buffer should be overwritten by https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-client/src/index.ts#L26-L28 if you use the lisk-client
Also, i have tested https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-client/index.html with this file.
it could be problem with the build process as described here https://github.com/LiskHQ/lisk-sdk/issues/5994

**Comment by shuse2** [comment]
this should be resolved by #6049
